ambassador/ambassador/config/config.py

```diff
@@ -63,6 +63,8 @@ class Config:
         'tracingservice': "tracing_configs",
     }
 
+    KnativeResources = ['ClusterIngress', 'kservice']
```

Comment:
How about a `set` instead of a `list` here?

Reply:
done